pages/k8s/metallb.md

```diff
@@ -34,19 +35,21 @@ anti-affinity to prevent Kubernetes pods from stacking on a single node.
 <div class="p-notification--positive is-inline">
   <div markdown="1" class="p-notification__content">
     <span class="p-notification__title">Note:</span>
-    <p class="p-notification__message">For more information on configuring MetalLB with Calico in BGP mode, please see this
-    <a href="https://metallb.universe.tf/configuration/calico/">
-    explanation of the required configuration</a> from the
-    <a href="https://metallb.universe.tf/"> MetalLB website</a></p>
+    <p class="p-notification__message">
+      For more information on configuring MetalLB with Calico in BGP mode, please see this
+      <a href="https://metallb.universe.tf/configuration/calico/">explanation of the required configuration</a> from the
+      <a href="https://metallb.universe.tf/"> MetalLB website</a>
+    </p>
   </div>
 </div>
+
 # Deployment
 
 ## Layer 2 mode
 
 The best way to deploy MetalLB in layer 2 mode on Charmed Kubernetes is with
 the MetalLB bundle, which includes Kubernetes operator charms both the
```

Comment:
```suggestion
the Kubernetes operator charms for both the
```

Reply:
I think we should move away from the bundle since the containers bundle is just leftover from the migration and we dont publish it anymore